providers/openebs/cspc_csi_upgrade/test_vars.yml

```diff
@@ -0,0 +1,14 @@
+# Test-specific parameters
+operator_ns: "{{ lookup('env','OPERATOR_NS') }}"
+test_name: cstor_cspc_csi_upgrade
+from_version: "{{ lookup('env','FROM_VERSION') }}"
+to_version: "{{ lookup('env','TO_VERSION') }}"
+upgrade_image_tag: "{{ lookup('env', 'UPGRADE_IMAGE_TAG') }}"
+upgrade_to_ci: "{{ lookup('env', 'UPGRADE_TO_CI') }}"
+cstor_operator: openebs-operator.yaml
```

Comment:
why we are downloading cstor_operator file in {{ path }}/openebs-operator.yaml ?